test/test.js

```diff
@@ -35,7 +40,10 @@ test('equals to sample', async t => {
 
 test('check internal structure', async t => {
   // Create deck as in previous example
-  const apkg = new AnkiExport('deck-name');
+  const apkg = new AnkiExport('deck-name', {
+    template,
+    DbClass: sql.Database
```

Comment:
I see. But as I understand - `test.js` test hole package. So I just change instantiating exporter module in test